src/docs/en/dev/zikula_sf_translator.md

```diff
@@ -0,0 +1,214 @@
+Zikula Translation's Guide
+=================
+
+This guide is intended to provide overview of translation technologies, usage, implementations and developement.
+
+## Table of contents
+
+* Introduction
+* Terminology
+* User's guide 
+	* Zikula legacy translator
+	* Zikula Symfony Translator
+* Translators
+* Developers guide
+	* Zikula legacy translator
+	* Symfony translator service
+	* Zikula Symfony Translator
+	* Testing
+* Important notes
+* See also sources
+
+
+
+## Introduction  
+Technology for translations used in this guide is simple - anywere in project we use English strings and English descriptions.
+These are translated to other language stored in files or database and loaded on demand instead of English strings. 
+
+## Terminology
+* **Symfony Translator** - Symfony gettext technology used for translations 
+* **Zikula legacy translator** - Zikula priror to 1.4.x uses own implementation of gettext translation technology. 
+* **Zikula Symfony Translator** - Extends Symfony Translator to support Zikula translation conventions. 
+* **message** -  in basic it is translation array element example: 'Englilsh string' => 'Translated string' 
+* **translation template** - .pot file for gettext used only in process of creating new translations. Not used in actual translating. 
+* **domain** - An optional way to organize messages into groups (e.g. Symfony admin, navigation or the default messages Zikula zikula, theme_themename, module_modulename and 2.0.0 bundlename)
+* **catalogue** - Gettext way to organize messages into groups (LC_MESSAGES, LC_TYPE, LC_ALL) 
+* **locale** - The locale that the translations are for (e.g. en_GB, en, etc);
+* **loader** - How Symfony should load and parse the file (e.g. xlf, php, yml, etc Zikula .po .mo).
+
+
+## User's guide
+Used technologies try to symplify translation process as possible. Installing language for Zikula is as simple as copying translation catalogue to aprioriate directory and enabling it in administration.
+You can find your locale translations on https://github.com/zikula-communities If there is no translation for your language and you willing to translate please check 'Translators' part of this guide. 
+### Zikula legacy translator
+
+Catalogue is always ``` LC_MESSAGES ```
+
+**Path to install translations on systems versions prior to 1.4.x**  
+
+``` Core translations /locale/catalogue/domain.loader ```
+
+``` Theme translations /themes/themename/locale/catalogue/domain.loader  where domain is 'theme_themename' ```
```

Comment:
@kaik - please update these paths so that variable values are shown differently than static values. Something like `/themes/{theme name}/{locale}/LC_MESSAGES/{domain}.{loader}` (or whatever is proper)